assets/src/dashboard/utils/useStoryView.js

```diff
@@ -87,13 +91,26 @@ export default function useStoryView({ filters, totalPages }) {
     [setPageClamped]
   );
 
+  const setPreviewVisible = useCallback(
+    (_, story) => {
+      if (enableStoryPreviews) {
+        _setPreviewVisible(story);
```

Comment:
Is there any danger in having the setter always exist even if the flag is off visually?

Reply:
Not unless I'm not thinking through something entirely. The only time the state gets updates is when the flag is true. So it's akin to having noops set for the onClick of the card containers but this way the feature flag is at the heart of the functionality.